ITEM.tsv

```diff
@@ -3506,8 +3506,8 @@ ITEM_20150317_003505	Hardened Tree Log
 ITEM_20150317_003506	Log required to fix the practice wood carving 
 ITEM_20150317_003507	Rope 
 ITEM_20150317_003508	Rope required to fix the scarecrow 
-ITEM_20150317_003509	Memory Fragment 
-ITEM_20150317_003510	An orb necessary to complete the spell that wakes the Sleeping Owls to aid Gaigalus. 
+ITEM_20150317_003509	$Fragment of Memory
```

Comment:
"Memory Fragment" is actually better. If you insist on changing it, then change it to "Fragment of **a** Memory"